server/apps/distributed-app/docs/modules/ROOT/pages/configure/index.adoc

```diff
@@ -52,6 +52,7 @@ By omitting these files, sane default values are used.
 ** xref:configure/recipientrewritetable.adoc[*recipientrewritetable.xml*] enables advanced configuration for the Recipient Rewrite Table component link:https://github.com/apache/james-project/blob/master/server/apps/distributed-app/sample-configuration/recipientrewritetable.xml[example]
 *** xref:configure/matchers.adoc[This page] allows choosing the indexing technology.
 ** xref:configure/usersrepository.adoc[*usersrepository.xml*] allows configuration of user storage link:https://github.com/apache/james-project/blob/master/server/apps/distributed-app/sample-configuration/usersrepository.xml[example]
+** xref:configure/droplists.adoc[This page] allows configuring drop lists.
```

Comment:
I would move this down into "extensions" as this is not enabled by default this likely isn't "core"